README.md

```diff
@@ -27,6 +27,62 @@ overlay network:
 * Step 2: Git clone the flannel repo: ```https://github.com/coreos/flannel.git```
 * Step 3: Run the build script: ```cd flannel; ./build```
 
+If you are building `flanneld` on CentOS 7, you may need to proceed with the following steps:
+
+1. Install Golang: 
+
+    ```
+    yum install -y git golang golang-vim golang-src
+    ```
+    
+2. If necessary, add proxy server addresses:
+
+    ```
+    export http_proxy="http://proxy.domain.com:9001"
+    export https_proxy="http://proxy.domain.com:9002"
+    ```
+
+3. Create a development directory and clone `flannel` project:
+
+    ```
+    cd ~/dev && git clone https://github.com/coreos/flannel.git && cd flannel
+    ```
+    
+4. Add `GOPATH` environment variable pointing to `flannel`'s workspace:
```

Comment:
Perhaps it is better to add the following code to `test`:

```
cd `dirname $0` || exit 1;

if [ -z "${GOPATH+x}" ]; then
        export GOPATH=${PWD}/gopath
fi

if [ -z "${GOBIN+x}" ]; then
        export GOBIN=${PWD}/bin
fi

go get golang.org/x/tools/cmd/cover
```

This way ${PWD} points to source code directory.